examples/plot_simulate_somato.py

```diff
@@ -94,45 +136,94 @@
+    location='proximal', synaptic_delays=synaptic_delays_p, seedcore=6)
+
+# Early distal drive
+weights_ampa_d = {'L2_basket': 0.0043, 'L2_pyramidal': 0.0032,
+                  'L5_pyramidal': 0.0009}
+weights_nmda_d = {'L2_basket': 0.0029, 'L2_pyramidal': 0.0051,
+                  'L5_pyramidal': 0.0010}
+synaptic_delays_d = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                     'L5_pyramidal': 0.1}
+
+net.add_evoked_drive(
+    'evdist1', mu=32., sigma=2.5, numspikes=1, sync_within_trial=True,
+    weights_ampa=weights_ampa_d, weights_nmda=weights_nmda_d,
+    location='distal', synaptic_delays=synaptic_delays_d, seedcore=6)
+
+# Late distal drive
+weights_ampa_d = {'L2_basket': 0.0041, 'L2_pyramidal': 0.0019,
+                  'L5_pyramidal': 0.0018}
+weights_nmda_d = {'L2_basket': 0.0032, 'L2_pyramidal': 0.0018,
+                  'L5_pyramidal': 0.0017}
+synaptic_delays_d = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                     'L5_pyramidal': 0.1}
+
 net.add_evoked_drive(
-    'evprox1', mu=20.0, sigma=0., numspikes=1, weights_ampa=weights_ampa_p,
-    weights_nmda=weights_nmda_p, location='proximal',
-    synaptic_delays=synaptic_delays_p, seedcore=6)
+    'evdist2', mu=84., sigma=4.5, numspikes=1, sync_within_trial=True,
+    weights_ampa=weights_ampa_d, weights_nmda=weights_nmda_d,
+    location='distal', synaptic_delays=synaptic_delays_d, seedcore=2)
 
-dpl = simulate_dipole(net, n_trials=1)
+###############################################################################
+# Now we run the simulation over 2 trials so that we can plot the average
+# aggregate dipole. As in :ref:`the MPIBackend example
+# <sphx_glr_auto_examples_plot_simulate_mpi_backend.py>`, we can use
+# ``MPIBackend`` to reduce the simulation time by parallizing across cells in
+# the network. However, no parallel backend is necessary. For a better
+# match to the empirical waveform, set ``n_trials`` to be >=25.
+n_trials = 2
+# n_trials = 25
+with MPIBackend(n_procs=2):
+    dpls = simulate_dipole(net, n_trials=n_trials)
 
-trial_idx = 0
-import matplotlib.pyplot as plt
-fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6))
-dpl[trial_idx].plot(ax=axes[0], show=False)
-net.cell_response.plot_spikes_hist(ax=axes[1])
-net.cell_response.plot_spikes_raster()
+###############################################################################
+# Finally, we plot the driving spike histogram, empirical and simulated median
+# nerve evoked response waveforms, and output spike histogram.
+fig, axes = plt.subplots(3, 1, sharex=True, figsize=(6, 6))
+net.cell_response.plot_spikes_hist(ax=axes[0],
+                                   spike_types=['evprox', 'evdist'],
+                                   show=False)
+axes[1].axhline(0, c='k', ls=':', label='_nolegend_')
+axes[1].plot(1e3 * stc.times, dipole_tc, 'r--')
+average_dipoles(dpls).plot(ax=axes[1], show=False)
+axes[1].legend(['MNE vertex', 'HNN simulation'])
```

Comment:
```suggestion
axes[1].legend(['MNE label average', 'HNN simulation'])
```

it's not "vertex" anymore.